tests/parametric/test_tracer.py

```diff
@@ -134,8 +134,8 @@ def test_tracer_commit_sha_environment_variable(
             },
         ],
     )
-    @missing_feature(context.library == "golang", reason="golang does not strip credentials yet")
-    @missing_feature(context.library == "nodejs", reason="nodejs does not strip credentials yet")
+    # @missing_feature(context.library == "golang", reason="golang does not strip credentials yet") # causing easy wins
```

Comment:
```suggestion
```

Reply:
should we just remove it altogether? I don't see any value in keeping it. Do we have to update the manifest/go.yaml file with the version that support was added?

Reply:
Deleted the comment and also updated the manifest file! thank you